packages/window-size-tool/rules.json

```diff
@@ -18,6 +18,7 @@
     "width": "> 768",
     "terms": [
       "//CLUSTER/@NUM_COLS = 2",
+      "//CLUSTER/@NUM_COLS = 2 and count(//CLUSTER[@NUM_COLS >= 2]//LIST) = 0 and count(//CLUSTER[@NUM_COLS > 2]) = 0",
```

Comment:
This new term needs to replace this term `"//CLUSTER/@NUM_COLS = 2",` because the rule will apply if _any_ term matches.

Reply:
oh my bad, will correct that now. Thanks Graham